service/apiserver/server.api

```diff
@@ -262,6 +262,7 @@ service server-api {
 
 type (
 	Tx {
+		Id             int64  `json:"id"`
```

Comment:
To be honest, exporting `Id` is kind of wired in blockchain. Let's think if there is better way.